website/screens/components/date-input/code/examples/uncontrolled.ts

```diff
@@ -4,8 +4,9 @@ const code = `() => {
   return (
     <DxcInset space="large">
       <DxcDateInput
+        size="fillParent"
         label="Uncontrolled"
-        helperText="The 'defaultValue' prop only works with uncontrolled date inputs"
+        helperText="Uncontrolled date input"
```

Comment:
The word **"Uncontrolled"** is three times in this example, I suggest at least removing one of them (maybe changing the helper text to "Helper text" or simply removing it).

Reply:
done